src/core/models/dt_ftrl.cc

```diff
@@ -970,8 +970,8 @@ dtptr Ftrl<T>::create_p(size_t nrows) {
   for (size_t i = 0; i < nlabels; ++i) {
     CString val;
     bool isvalid = col0_str64.get_element(i, &val);
-    labels_vec[i] = isvalid? std::string(val.ch, static_cast<size_t>(val.size))
-                           : std::string();
+    labels_vec[i] = val.to_string();
+    xassert(isvalid == !val.isna());  (void)isvalid;
```

Comment:
For invalid elements, do we always return `val` that has `ptr_ == nullptr`? Or there could be some garbage?

Reply:
You're right, there could be garbage (as with all other types).

Reply:
I fixed it